app/src/main/java/org/mozilla/firefox/firefoxlitedownloader/MainActivity.java

```diff
@@ -1,50 +1,476 @@
+        progressBarTwo = findViewById(R.id.progressBarTwo);
+        progressBarThree = findViewById(R.id.progressBarThree);
+        progressBarFour = findViewById(R.id.progressBarFour);
+        progressBarFive = findViewById(R.id.progressBarFive);
+        progressBarSix = findViewById(R.id.progressBarSix);
+        progressBarSeven = findViewById(R.id.progressBarSeven);
+        progressBarEight = findViewById(R.id.progressBarEight);
+        progressBarNine = findViewById(R.id.progressBarNine);
+        progressBarTen = findViewById(R.id.progressBarTen);
+        progressBarEleven = findViewById(R.id.progressBarEleven);
+        progressBarTwelve = findViewById(R.id.progressBarTwelve);
+        progressBarThirteen = findViewById(R.id.progressBarThirteen);
+        progressBarFourteen = findViewById(R.id.progressBarFourteen);
+        progressBarFifteen = findViewById(R.id.progressBarFifteen);
+    }
+    public void displaynotification(View view){
+        final NotificationCompat.Builder builder= new NotificationCompat.Builder(this);
+        final int notifyid=1;
+        final NotificationManager nm=(NotificationManager) getSystemService(Context.NOTIFICATION_SERVICE);
+        new Thread(new Runnable() {
             @Override
-            public void onClick(View v) {
-                MainFunctionAccessFile accessFile=new MainFunctionAccessFile();
-                accessFile.download();
+            public void run() {
+                int icr;
+                for(icr=0;icr<=100;icr+=5){
+                    builder.setProgress(100,icr,false);
+                }
+            }
+        });
+
+    }
+    public void onClickListenerOne() {
+        buttonOne.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                Log.d("jjk",MozzDownloader.getStatus(downloadIdOne)+"");
+//                if (Status.RUNNING == MozzDownloader.getStatus(downloadIdOne)) {
+//                    MozzDownloader.pause(downloadIdOne);
+//                    return;
+//                }
+                if (Status.QUEUED == MozzDownloader.getStatus(downloadIdOne)) {
+                    MozzDownloader.pause(downloadIdOne);
+                    return;
+                }
 
+                buttonOne.setEnabled(false);
+                progressBarOne.setIndeterminate(true);
+                progressBarOne.getIndeterminateDrawable().setColorFilter(
+                        Color.BLUE, android.graphics.PorterDuff.Mode.SRC_IN);
+
+                if (Status.PAUSED == MozzDownloader.getStatus(downloadIdOne)) {
+                    MozzDownloader.resume(getApplicationContext(),mainActivity,downloadIdOne);
+                    return;
+                }
+
+                downloadIdOne = MozzDownloader.download(URL1, dirPath, "mandys.pmd")
+                        .build()
+                        .setOnStartOrResumeListener(new OnStartOrResumeListener() {
```

Comment:
will MozzDownloader now have the reference to this anonymous class (which is an Activity)?